vividus-to-xray-exporter/src/main/java/org/vividus/xray/databind/ManualTestCaseSerializer.java

```diff
@@ -105,8 +107,11 @@ private static void writeJsonArray(JsonGenerator generator, String startField, C
     private static void writeObjectWithField(JsonGenerator generator, String objectKey, String fieldName,
             String fieldValue) throws IOException
     {
-        generator.writeObjectFieldStart(objectKey);
-        generator.writeStringField(fieldName, fieldValue);
-        generator.writeEndObject();
+        if (fieldValue != null)
```

Comment:
shouldn't you put this check on the particular field? this change is not safe since it could introduce bugs when mandatory fields are missed

Reply:
At some point it looks like double check because we already have jira that will fail if mandatory fields are missed